runtime/common/Cargo.toml

```diff
@@ -42,6 +43,9 @@ pallet-pools = { path = "../../pallets/pools", default-features = false }
 common-types = { path = "../../libs/common-types", default-features = false }
 pallet-permissions = { path = "../../pallets/permissions", default-features = false }
 
+[dev-dependencies]
+sp-io = { git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.24" }
+
 [features]
```

Comment:
I believe we will need to add `pallet-treasury/std` here or else `runtime-common` won't compile when building altair/centrifuge when this branch is merged and the full CI will attempt to build the runtimes.

Reply:
The `pallet-treasury/str` is already imported under the `[dependencies]` tag. I do not know if there is something more I'm not seeing regarding it...

Reply:
Sorry, I messed up where I wanted to place this comment. I mean that we need to add `pallet-treasury/std` to the `std = ` list/

Reply:
Ohh, ok that makes much more sense 😃. You're right!